Homework8/Task2.py

```diff
@@ -0,0 +1,14 @@
+word = input("Enter the word: ")
+vowels = "aeiou"
+
+
+for i in word:
+    is_vowel = False
+    for j in vowels:
+        if i == j:
+            is_vowel = True
+            break
+    if not is_vowel:
+        print(i, end = " ")
```

Comment:
ამოცანა სწორად არის გადაჭრილი, 

იხილეთ ლაკონური ვერსია

```suggestion
    if i not in vowels:
        print(i, end = " ")
```